### PR TITLE
Add doc for statistics-service

### PR DESCRIPTION
**Pull request by StackScribe**
Signed-off-by: Meg McRoberts <meg.mcroberts@dynatrace.com>

https://github.com/keptn/keptn.github.io/issues/1003 

This copies the content of https://github.com/keptn/keptn/blob/master/statistics-service/README.md to keptn.io with some small edits.

### CODE REVIEW

File: content/docs/concepts/statistics/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+http://keptn-api-url.com/api/statistics
+```
+
+Use the following URL to view the swagger UI for the service:
+
+```
+http://keptn-api-url.com/api/swagger-ui/?urls.primaryName=statistics
+```
+
+To browse the API docs, open the Swagger docs in your [browser](http://localhost:8080/swagger-ui/index.html).
```
Comment (StackScribe) [inline content/docs/concepts/statistics/index.md:45]: Apparently this is a link to the local docs but it is a broken link if one is reading this on a system that is not running Keptn with the Statistics Service configured.  How can it be reworded?

Comment (oleg-nenashev) [reply content/docs/concepts/statistics/index.md:45]:
```suggestion
To browse the API docs, open the Swagger docs in your browser, e.g. `${KEPTN_URL}/swagger-ui/index.html`.
```

File: content/docs/concepts/statistics/index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+---
+title: Statistics Service
+description: Provide usage statistics about a Keptn installation and optionally display them on a Swagger dashboard.
+
+weight: 10
+icon: setup
```
Comment (oleg-nenashev) [inline content/docs/concepts/statistics/index.md:6]: Looks okay for now

**Review by oleg-nenashev — COMMENTED** [review]
I suggest submitting the URL fix to upstream too. Otherwise looks good, including the selected weight

File: content/docs/concepts/statistics/index.md (hunk 1 above)
Comment (robertjahn) [reply content/docs/concepts/statistics/index.md:45]: In my environment - the URL is `/api/swagger-ui/`  -- it has the API in the path and index.html is not required to list.

Comment (StackScribe) [reply content/docs/concepts/statistics/index.md:45]: When we figure out how to resolve this issue here, I will generate a PR for https://github.com/keptn/keptn/blob/master/statistics-service/README.md  to fix the problem there until we de-duplicate the README text.

**Review by oleg-nenashev — APPROVED** [review]
I think it is net improvement. Let's get it merged

**Pull request merged**